bigbluebutton-html5/imports/ui/components/user-list/component.jsx

```diff
@@ -102,57 +65,56 @@ class UserList extends Component {
         break;
     }
 
-    if (Args[0].keyCode === KEY_CODES.ESCAPE
+    if (event.keyCode === KEY_CODES.ESCAPE
```

Comment:
Could you comment on all those conditions? Is hard to understand what `counter === -1` means or `counter > count`.